alf/networks/transformer_networks.py

```diff
@@ -84,6 +85,10 @@ def __init__(self,
             return_core_only (bool): If True, only return the core embedding.
                 Otherwise, return all embeddings
             core_size (int): size of core (i.e. number of embeddings of core)
+            use_core_embedding (bool): whether to use learnable core embedding.
```

Comment:
What's the purpose/benefit of learnable core embedding?

Reply:
As the docstring of this class says, it is same as [CLS] in BERT model, which is used to generate a fixed dimension embedding for a sequence.